docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -32,154 +32,154 @@ Added a new optional parameter `options`, which is a dictionary with one valid v
 
 Added a new object that represents the content of an attachment.
 
-**Available in**: Outlook 2019 for Windows (Office 365 subscription)
+**Available in**: Outlook for Windows (Office 365)
```

Comment:
Are you sure this change is accurate? Originally this line said available in "Outlook 2019 for Windows" -- which isn't necessarily the same as "Outlook for Windows".

Reply:
(same comment applies to all instances of this change throughout this article)

Reply:
Yes, I'm sure. Also, I'm trying to make the naming of Office 365 subscription version vs one-time purchase version more consistent. In this case, there should be no preview items in one-time purchase of Office/Outlook.